app/src/androidTest/java/com/github/wnder/NetworkInformationTest.java

```diff
@@ -0,0 +1,26 @@
+package com.github.wnder;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.wifi.WifiManager;
+import android.provider.Settings;
+
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class NetworkInformationTest {
+    @Test
+    public void isNetworkAvailableWorksWhenThereIsConnection(){
+        Context context = ApplicationProvider.getApplicationContext();
+
+        assertThat(NetworkInformation.isNetworkAvailable(context), Matchers.is(true));
+    }
+}
```

Comment:
This test seems incomplete. Did you try to remove the internet connection of the emulator?

Reply:
Yes I tried. At some point in time, you could active the airplane mode but they removed the functionality. I have not found any methods that could do that now.

Reply:
Ok, that's fine.